packages/paste-website/src/components/homepage/HomeHero.tsx

```diff
@@ -74,19 +74,18 @@ const HomeHero: React.FC = () => {
                 <Badge as="span" variant="new">
                   New!
                 </Badge>
-                <NewComponentBannerText>We&apos;ve released a Figma plugin, Twilio Ipsum!</NewComponentBannerText>
+                <NewComponentBannerText>We&apos;ve updated our Design tokens list page!</NewComponentBannerText>
```

Comment:
suggestion (non-blocking): a little dangling word on our large screen size: 
![image](https://user-images.githubusercontent.com/1592327/186249954-0a268201-7950-40fd-8dee-616309a54507.png)

```suggestion
                <NewComponentBannerText>We updated our tokens list page!</NewComponentBannerText>
```